### PR TITLE
Add sample for CRUD on GCS.

### PR DESCRIPTION
**Pull request by jerjou**
(or at least, CRD)

### CODE REVIEW

**Comment by jonparrott** [comment]
Travis doesn't like this, can you see why?

**Comment by jerjou** [comment]
....partly because past-jerjou forgot to actually run the tests -_-;

But partly also because the `cloud_sql` test can't find `MySQLdb`? Seems unrelated.. do you know about that?

**Comment by jerjou** [comment]
oh wait. my bad. wait a sec...

**Comment by jerjou** [comment]
Tests passing. PTAL

File: storage/api/crud_object.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    # http://g.co/cloud/storage/docs/json_api/v1/objects/insert#request
+    body = {
+        'name': filename,
+    }
+
+    # If specified, create the access control objects and add them to the
+    # request body
+    if readers or owners:
+        body['acl'] = []
+    for r in readers:
```
Comment (jonparrott) [inline storage/api/crud_object.py]: nit: can you put a newline between control structures?

**Comment by jonparrott** [comment]
LGTM with tiny nit. Feel free to merge once that's good.

**Pull request merged**